react/src/component/AuthView.jsx

```diff
@@ -554,6 +557,31 @@ export default class AuthView extends BaseView {
         }).catch(this.snackBarError)
     }
 
+    downloadAgentConfigFile(agentId) {
+        axios({
+            url: `/api/agent/downloadAgentConfigFile/${agentId}`,
+            method: 'GET',
+            responseType: 'blob'
+        }).then((res) => {
+            const href = URL.createObjectURL(res.data);
```

Comment:
When the back-end returns result.error, the application.yml would be like that.
![image](https://user-images.githubusercontent.com/26757995/212269468-86bff3b0-8af5-4d8f-aa7e-8099474ea9bf.png)
We need to check the type of res.data, prase res.data to json and send message to the bar. 
![image](https://user-images.githubusercontent.com/26757995/212271259-9cb3f5c7-5931-4d2b-86d1-f62d1a03a4fb.png)

For example, 
```
if (res.data.type.includes('application/json')) {  
    let reader = new FileReader()
    reader.onload = function () {
       let result = JSON.parse(reader.result) 
       if (result.Code !=  200) { 
           .......
        }
     }
     reader.readAsText(res.data)
}else{
   download file....
}
```